crates/voicevox_core/src/status.rs

```diff
@@ -76,7 +76,7 @@ impl Status {
             merged_metas: VoiceModelMeta::default(),
             light_session_options: SessionOptions::new(cpu_num_threads, false),
             heavy_session_options: SessionOptions::new(cpu_num_threads, use_gpu),
-            id_relations: BTreeMap::default(),
+            id_relations: BTreeMap::new(),
```

Comment:
```suggestion
            id_relations: BTreeMap::default(),
```

今ここら辺を改革するPRを出しており、後でコンフリクトが発生すると思います。

Reply:
なんか二重にreview commentが発生した...